### PR TITLE
set up of associations

### PR DESCRIPTION
**Pull request by hasnatali1947**
- Run a program using the command line.
- Set up associations between classes and objects.

### CODE REVIEW

File: classroom.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :label, :student
```
Comment (DeliceLydia) [inline classroom.rb]: - [x] I suggest that you make `student` an `attr_reader` only

File: students.rb [hunk 2]
```diff
@@ -1,14 +1,20 @@
 require_relative './person'
+require_relative './classroom'
 
 class Student < Person
   attr_accessor :classroom
```
Comment (DeliceLydia) [inline students.rb]: - [ ] I suggest that you make `classroom` an `attr_reader` as well we only want to read it

File: book.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+require_relative 'rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```
Comment (DeliceLydia) [inline book.rb]: - [x] Please make `rentals` an `attr_accessor` we want to read it and modify it

File: person.rb [hunk 4]
```diff
@@ -1,18 +1,26 @@
 require_relative 'nameable'
+require_relative 'rental'
+require_relative 'book'
 
 class Person < Nameable
-  attr_reader :id, :parent_permission
   attr_accessor :name, :age
+  attr_reader :id, :rentals
```
Comment (DeliceLydia) [inline person.rb]: - [x] Please make `rentals` an `attr_accessor` as well

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @hasnatali1947, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Followed git workflow
- Created the remaining classes for the school library 
- Created associations between them.


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classroom.rb [hunk 5]
```diff
@@ -0,0 +1,16 @@
+require_relative 'student'
+
+class Classroom
+  attr_accessor :label
+  attr_reader :student
```
Comment (julie-ify) [inline classroom.rb:5]: - [ ] **[OPTIONAL]** This should be `students`, not `student`.

**Review by julie-ify — APPROVED** [review]
Hi @hasnatali1947,

## Highlights!
- Good code organization and associations ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**